src/Autofac.BindingByConvention/FluentSyntax/IFluentExceptTypes.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Autofac.BindingByConvention.FluentSyntax
+{
+    using System;
+
+    public interface IFluentExceptTypes
+    {
+        bool CheckAllFiltersSatisfied(Type implementationType);
```

Comment:
Dans l'implémentation, tu jettes une exception `throw new NotImplementedException();`
Du coup pourquoi exposer dès maintenant cette méthode ?

Reply:
because it is called by the rest of the fluent syntax plumbing.

Reply:
... I added the implementation, it was rather quick.